ARCHITECTURE.md

```diff
@@ -58,7 +58,7 @@ the local hard drive.
 
 `Resolve` is the representation of a directed acyclic graph of package
 dependencies, which uses `PackageId`s for nodes. This is the data
-structure that is saved to the lock file. If there is no lockfile,
+structure that is saved to the lock file. If there is no lock file,
```

Comment:
Do we prefer "lockfile" or "lock file"?

Reply:
I just normalised to the more common variant; otherwise ambivalent.

Reply:
Yeah I can believe that.  I _think_ I've been using lockfile, but I'm happy with a decision and applying it uniformly.